arbitrator/config/arbitrator_params.yaml

```diff
@@ -6,7 +6,7 @@ min_plan_duration: 6.0
 # Float: The nominal amount of time that an arbitrated plan should cover for the 
 # system to operate at best performance
 # Unit: s
-target_plan_duration: 15.0
+target_plan_duration: 13.0
```

Comment:
Why is this 13 now? Seems like 15 was just fine and I noticed another 15 parameter below which might need to match

Reply:
Accepted per our discussion on Slack.